base/build.json

```diff
@@ -1,5 +1,6 @@
 {
   "image": "homeassistant/{arch}-homeassistant-base",
+  "picotts_hash": "e3ba46009ee868911fa0b53db672a55f9cc13b1c",
```

Comment:
An example how it should look like: https://github.com/home-assistant/hassio-addons/blob/master/zwave/build.json
 - Looking also into https://github.com/home-assistant/hassio-addons/blob/master/zwave/Dockerfile to understand how the Dockerfile should be formated

Reply:
Rebasing took place. Update and CI/CD in progress.

Thanks for the advice.